xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -232,21 +232,22 @@
 #panelheader($services.localization.render('panels.applications.title'))
 
 {{html}}
-&lt;ul class="applicationsPanel nav nav-pills nav-stacked"&gt;
+&lt;ul class="applicationsPanel nav"&gt;
   ## Get the blacklist configuration
   #set ($configDoc = $xwiki.getDocument($services.model.createDocumentReference('', 'PanelsCode',
     'ApplicationsPanelConfiguration')))
   #displayAppPanelEntries('org.xwiki.platform.panels.Applications' $configDoc)
 &lt;/ul&gt;
-&lt;ul class="applicationsPanel applicationsPanelMoreList nav nav-pills nav-stacked"&gt;
+&lt;ul class="applicationsPanel applicationsPanelMoreList nav"&gt;
   &lt;li&gt;
-    &lt;a href="$xwiki.getURL($services.model.createDocumentReference('', 'XWiki', 'XWikiPreferences'), 'admin',
-        'editor=globaladmin&amp;section=XWiki.AddExtensions')" class="applicationPanelMoreButton" title="$services.localization.render('panels.applications.more')"&gt;
+    #set ($panelId = $doc.getXDOM().getIdGenerator().generateUniqueId('applicationPanelMoreContainer',''))
```

Comment:
I hadn't noticed that before. This seems wrong. `$doc.getXDOM()` returns a clone of a cached XDOM instance, I don't think it makes sense to use its id generator.

Reply:
@michitux 
I'm not sure this is possible to keep the XDOM object through different calls of this panel.
I can't find how to access a proper IdGenerator here.
Instead, we could append a random alphanumeric string at the end of the string to make it 'unique'.

Change proposed in https://github.com/xwiki/xwiki-platform/pull/2157/commits/926c9109b2023237c74aa4d740fe489dcdd7d2e3
![20843-UniqueID](https://github.com/xwiki/xwiki-platform/assets/28761965/800dcbd4-41bc-44a3-a989-dc104dfef04f)
Note that this is very similar to the solution used before (10 alphanumeric instead of a number from 1 to 100, but the concept is the same).